sos/report/plugins/qdevice.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2024 Javier Blanco <javier@jblanco.es>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, IndependentPlugin
+
```

Comment:
flake is reporting an issue  here, we need to add a new line here please

```
sos/report/plugins/qdevice.py:13:1: E302 expected 2 blank lines, found 1
```

Reply:
Added in commit bd60aebbebd9b30780784489db4b77f01c0b02d9